posts/Yoann/2018-11-29-hibernate-search-6-0-0-Alpha1.adoc

```diff
@@ -0,0 +1,179 @@
+= Get a sneak peek into the future of Hibernate Search with 6.0.0.Alpha1!
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "Lucene", "Elasticsearch", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+We just published Hibernate Search 6.0.0.Alpha1,
+the first release for the next major version of Hibernate Search.
+
+This version is **not suitable for production** due to its limitations and to the fact APIs are still unstable.
+However, it's a great way to get a feeling of where Hibernate Search is going and give us your feedback.
+
++++<!-- more -->+++
+
+== What's new
+
+For a full list of changes since the previous releases,
+please see the https://hibernate.atlassian.net/secure/ReleaseNote.jspa?projectId=10061&version=31657[release notes].
+Be warned, though: there is a lot in there. You may find the summary below easier to process.
+
+=== Preview status
+
+Hibernate Search 6.0 is currently a *technology preview* and is *not ready for production*.
+
+While the core of Hibernate Search 6 is there, there are still a lot of limitations:
+
+* Features are partially implemented or simply missing.
+For example, you cannot index a `Double` property yet,
+you cannot configure custom Lucene directories,
+you cannot ignore bridges in the predicate DSL,
+and so on.
+* APIs are not considered stable yet.
+* The low-level parts of Hibernate Search 5 that made its efficiency haven't been
```

Comment:
"made its efficiency" ?

Reply:
"made it efficient" maybe?

Reply:
what about `... Search 5 that enhances (or improves) efficiency ..`

Reply:
"improve its efficiency" it is. Thanks